src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -85,15 +85,15 @@ const deriveEnrollmentType =
           .map(({ status, lastUpdated }) => ({ status, lastUpdated }));
 
 
-      const { ACTIVE, CANCELLED, COMPLETED, NOT_ENROLLED } = enrollmentTypes;
+      const { ACTIVE, CANCELLED, COMPLETED } = enrollmentTypes;
```

Comment:
Realised that there is no use case  for `NOT_ENROLLED`. 
This is because at the moment we are only fetching TEInstances from either a context of 
a) Program (Tracker & Event) or 
b) TEType. 

In case a) we have `ACTIVE`, `CANCELLED`, `COMPLETED`  and in case b) we have `DONT_SHOW_TAG`